.eslintrc.js

```diff
@@ -123,6 +123,14 @@ module.exports = {
 						importNames: [ 'combineReducers' ],
 						message: "`combineReducers` should be imported from 'state/utils', not 'redux'.",
 					},
+					// Deprecate createReducer and createReducerWithValidation.
+					{
+						name: 'state/utils',
+						importNames: [ 'createReducer', 'createReducerWithValidation' ],
+						message:
+							'This method is deprecated; please use a plain reducer function ' +
```

Comment:
It might be nice to set 2 individual rules so the message can be very specific depending on the import:

- Use a reducer function (`switch` statement)
- Use a reducer function wrapped in `withSchemaValidation`.

Reply:
I did try that, but unfortunately, since both rules use the same `name`, the second one ended up rewriting the first :-/ I could be missing something, of course.

Reply:
That's disappointing, not a blocker if it's a limitation of the rule 👍